### PR TITLE
modify following files

### PR DESCRIPTION
**Pull request by leonchen83**
pom
package name
git ignore
readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,17 +13,17 @@ This project is currently WIP and the interface may change. Also note that only
...
-    3. `mvn -Dmaven.test.skip=true install`
+    3. `mvn clean install -Dmaven.test.skip=true`
 
 ## Usage example
 
-```java
+```java  
 import redis.clients.jedis.Jedis;
-import io.rejson.JReJSON;
+import com.redislabs.modules.rejson.JReJSON;
```
Comment (itamarhaber) [inline README.md]: I don't know - naming things is hard... @dvirsky @dragnot - wdut?

File: .gitignore [hunk 2]
```diff
@@ -4,3 +4,25 @@
...
+
+# virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
+hs_err_pid*
+
+# Eclipse
+/.project
+/.classpath
+
+# Idea
+/redis-replicator.iml
```
Comment (itamarhaber) [inline .gitignore]: This line looks a bit out of place :)

File: README.md (hunk 1 above)
Comment (leonchen83) [inline README.md]: basically, package name same as maven group name. so I do above change

File: .gitignore (hunk 2 above)
Comment (leonchen83) [inline .gitignore]: copy that file and miss modify name :)

**Comment by itamarhaber** [comment]
Thanks so much - merging.

**Pull request merged**